media/css/firefox/whatsnew/whatsnew-96-de.scss

```diff
@@ -0,0 +1,89 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo-product-relay';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+@import '~@mozilla-protocol/core/protocol/css/components/picto';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/templates/multi-column';
+@import 'includes/base';
+@import 'includes/dark-mode';
+@import '../../base/mozilla-fxa-state';
+
+// * -------------------------------------------------------------------------- */
+// Main content
+
+.wnp-content-main {
+    padding: 0;
+    text-align: center;
+
+    .mzp-l-content {
+        padding-bottom: $layout-sm;
+    }
+}
+
+.wnp-main-logo {
+    margin-bottom: $spacing-xl;
+}
+
+.logo-lockup {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: $spacing-xl;
+
+    span {
+        @include font-size(24px);
```

Comment:
**Question:** why not write `font-size: 24px;` instead? I thought we only used `@includes` when using Protocol's pre-defined font-sizes. However, looking at more Bedrock code, I notice that the team has in fact used the former in other pages, too.

Reply:
The `font-size()` mixin converts `px` units to `rem` units (with a `px` fallback for older browsers). Using relative units is better for accessibility, since it will scale with users default font sizes should they pick a custom value e.g. for better readability). We should ideally be using the `font-size()` mixin everywhere.